string.md

```diff
@@ -17,9 +17,9 @@ In CockroachDB, the following are aliases for `STRING`:
 
 ## Length
 
-At this time, length is always variable for a `STRING` value. If you specify a fixed length by declaring `CHAR(n)` or `VARCHAR(n)` on a column, it will not be enforced.
+To limit the length of a string column, use `CHAR(n)` or `VARCHAR(n)`, where `n` is the maximum number of characters allowed.
 
-{{site.data.alerts.callout_info}}A future version of CockroachDB will support enforcing length limits on strings.{{site.data.alerts.end}}
+When inserting a string value, if the value exceeds the column's length limit, Cockroach gives an error.
```

Comment:
You should also note that when casting to a string with a length limit, truncation is performed:

```
root@:26257> select 'hello world'::char(5);
+--------------------------------+
| CAST('hello world' AS CHAR(5)) |
+--------------------------------+
| hello                          |
+--------------------------------+
```